department-of-veterans-affairs/caseflow
# Allow all orgs to bulk assign tasks

Resolves #14124

### Description
Gives all orgs the ability to bulk assign tasks

### Acceptance Criteria
- [ ] All orgs can bulk assign

### Testing Plan
1. Log in a sjolly postman and go to the mail team queue
1. Confirm they have the bulk "Assign tasks" button
1. Click "Assign tasks"
1. Assign some tasks to a user
1. Confirm tasks were assigned to the user.

### User Facing Changes
 - [ ] All orgs now have bulk assign modal

 BEFORE|AFTER BUTTON|AFTER MODAL
 ---|---|---
![Screen Shot 2020-06-10 at 11 29 33 AM](https://user-images.githubusercontent.com/45575454/84287260-adfa2d80-ab0d-11ea-9639-b5556a833a2b.png)|![Screen Shot 2020-06-10 at 4 17 11 PM](https://user-images.githubusercontent.com/45575454/84314348-e4e43980-ab35-11ea-9f97-87f5a50ed117.png)|![Screen Shot 2020-06-10 at 4 17 19 PM](https://user-images.githubusercontent.com/45575454/84314355-e877c080-ab35-11ea-8318-1e9bd4aa4d4a.png)



AFTER VALIDATION|AFTER ASSIGNEE|AFTER REGIONAL OFFICE|AFTER TASK TYPES|AFTER TASK COUNT
---|---|---|---|---
![Screen Shot 2020-06-10 at 11 47 00 AM](https://user-images.githubusercontent.com/45575454/84289178-2cf06580-ab10-11ea-99c5-80fd751d8d77.png)|![Screen Shot 2020-06-10 at 11 47 08 AM](https://user-images.githubusercontent.com/45575454/84289188-2eba2900-ab10-11ea-9cfa-901345b7d43d.png)|![Screen Shot 2020-06-10 at 11 47 13 AM](https://user-images.githubusercontent.com/45575454/84289193-2feb5600-ab10-11ea-972e-c0560bd87124.png)|![Screen Shot 2020-06-10 at 11 47 17 AM](https://user-images.githubusercontent.com/45575454/84289198-31b51980-ab10-11ea-990e-9246336d4b18.png)|![Screen Shot 2020-06-10 at 11 47 22 AM](https://user-images.githubusercontent.com/45575454/84289202-32e64680-ab10-11ea-9401-8c2405728177.png)


### Code Documentation Updates
